pkg/limayaml/limayaml.go

```diff
@@ -319,3 +320,18 @@ type CACertificates struct {
 	Files          []string `yaml:"files,omitempty" json:"files,omitempty" jsonschema:"nullable"`
 	Certs          []string `yaml:"certs,omitempty" json:"certs,omitempty" jsonschema:"nullable"`
 }
+
+// Engine is the name of the container engine, e.g. "docker", "containerd", that contains a list of sockets to monitor.
```

Comment:
This comment doesn't sound correct (`Engine` is not a name).

```suggestion
// Engine is a list of container engine connection details.
```